### DISCUSSION TITLE
How to add clusters?

### DISCUSSION THREAD

**Discussion by spyshower** [rnmapbox/maps#1426] [Q&A]
This is not a bug report.

How to add clusters to the map? I understand that the whole thing is not a responsibility of this package, but rather the ancient mapbox one.

I tried to search the old internet (including in this repo's issues' section) but all I could find is some code that uses `MapboxGL.geoUtils.makeFeatureCollection()` but this function returns undefined error.

**Comment by jaltin** [comment]
Check out this example that does clustering: https://github.com/react-native-mapbox-gl/maps/blob/master/example/src/examples/EarthQuakes.js

Also, you an read about it in the documentation: https://github.com/react-native-mapbox-gl/maps/blob/master/docs/ShapeSource.md

In general, issues are supposed to be used for bugs and problem solving. I suggest you post on stack overflow instead when you need help with how to do things.
